utils/src/experiment.ts

```diff
@@ -31,8 +31,6 @@ export interface CohortParticipantConfig {
   minParticipantsPerCohort: number|null;
   // Max number of participants for experiment cohort (or null if no limit)
   maxParticipantsPerCohort: number|null;
-  // If false, exclude booted participant from min/max participant counts
-  includeAllParticipantsInCohortCount: boolean;
```

Comment:
Do you think we might ever need this in the future? If so, it may be best to keep the field in (and just not use it / have it default to false) to avoid backwards compatibility issues later.

Reply:
Sounds good, will leave this field in and just hide it in the experiment creation.